js/admin.js

```diff
@@ -7,6 +7,23 @@
 			mode: 'shell'
 		} );
 
+	function checkForAdsFile(){
+		var currentTime = (new Date()).getTime();
+		var adstxtUrl = '/ads.txt?currentTime=' + currentTime;
+
+		$.get(adstxtUrl, function( data, status ){
+			$('.existing-adstxt').show();
+		}).fail(function(){
+			// Ads.txt not found
+			$('.existing-adstxt').hide();
+		});
+	}
+	
+	// Call our check when we first load the page
+	checkForAdsFile();
+
+	$('.ads-txt-rerun-check').click( checkForAdsFile );
```

Comment:
keep this consistent with other handlers
```suggestion
	$( '.ads-txt-rerun-check' ).on( 'click', checkForAdsFile );
```